Docs/Configuration.tex

```diff
@@ -1576,12 +1560,12 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
 
   Some modern firmware, including on virtual machines such as VMware,
   may update pointers to UEFI services during driver loading and related actions.
-  Consequentially this directly breaks other quirks that affect memory management,
+  Consequently, this directly breaks other quirks that affect memory management,
   such as \texttt{DevirtualiseMmio}, \texttt{ProtectMemoryRegions}, or \texttt{RebuildAppleMemoryMap},
-  and may also break other quirks depending on the effects of these.
+  and may also break other quirks depending on the scope of such.
```

Comment:
Prefer original, here (and change possibly changes meaning? i.e. effects -> scope).

Reply:
Let Vit weigh in

Reply:
cc @vit9696

Reply:
other quirks as well in my opinion. Hard to predict or explain.